rust/src/treefile.rs

```diff
@@ -574,6 +590,34 @@ impl Treefile {
         self.parsed.packages.clone().unwrap_or_default()
     }
 
+    pub(crate) fn get_packages_local(&self) -> Vec<String> {
+        self.parsed
+            .derive
+            .packages_local
+            .iter()
+            .flatten()
+            .map(|(k, v)| format!("{}:{}", v, k))
```

Comment:
One thing that's going to be awesome is when we start moving some of `rpmostree-core.cxx` to Rust, because then lots of these accessor methods become redundant - we can natively use Rust data types, etc.